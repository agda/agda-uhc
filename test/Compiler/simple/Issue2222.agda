open import Common.IO

header = putStrLn "foo"

main : IO _
main = putStrLn "42"
