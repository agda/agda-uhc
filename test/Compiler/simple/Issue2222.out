EXECUTED_PROGRAM

ret > ExitSuccess
out > 42
out >
